README.md

```diff
@@ -111,8 +92,8 @@ npm run dev
 Open [http://localhost:3000](http://localhost:3000) with your browser to see
 the result. If you have followed the
 [backend development setup instructions](https://github.com/BLSQ/openhexa-app/#local-development),
-you should be able to login with the listed credentials in those instructions:
-`root@openhexa.org`/`root` (please check if it hasen't been updated).
+you should be able to log in with the listed credentials in those instructions:
+`root@openhexa.org`/`root` (please check if it hasn't been updated).
```

Comment:
I'd add a link to openhexa-app to let the reader check if the credentials haven't been updated.

Reply:
Good idea, done!